azure_arc_data_jumpstart/microk8s/azure/arm_template/VNET.json

```
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-08-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetworkName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the VNET"
+      }
+    },
+    "subnetName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the subnet in the virtual network"
+      }
+    },
+    "addressPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "VNET CIDR"
+      }
+    },
+    "subnetAddressPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Subnet CIDR"
+      }
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Network/virtualNetworks",
+      "comments": "Deploys a VNET and Subnet for Sidecar and K8s VM",
```

```suggestion
      "comments": "Deploys a VNET and Subnet for Client and K8s VM",
```

We are keeping the "Client" wording for now and we will re-visit the sidecar terminology in the future. Make sure to update wherever needed as well as screenshots.

---

Agreed, updated markdown and Screenshot.